test/e2e/resize_nodes.go

```diff
@@ -361,10 +361,15 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 		}
 	}()
 
+	Logf("Waiting for node %s to be ready", node.Name)
```

Comment:
OK, so what you're doing is ensuring that the node is ready before the test actually begins. Do you think it would be better to move this outside of this function and just do it above the function call? I'd do that, and change the logging statement to make crystal clear what you're doing so idiots like me don't get confused (e.g. "waiting %v to ensure node %s is ready before beginning %s test...")